src/js/view/week/allday.js

```diff
@@ -65,29 +70,49 @@ util.inherit(Allday, View);
 Allday.prototype.render = function(viewModel) {
     var container = this.container;
     var scheduleContainerTop = this.options.scheduleContainerTop;
-    var weekdayView;
+    var aboutMe = {
+        panelName: PANEL_NAME,
+        forcedLayout: this.forcedLayout
+    };
     var self = this;
+    var weekdayView;
 
     container.innerHTML = tmpl(this.options);
 
     this.children.clear();
 
     weekdayView = new WeekdayInWeek(
         this.options,
-        domutil.find(config.classname('.weekday-container'), container)
+        domutil.find(config.classname('.weekday-container'), container),
+        aboutMe
     );
+    weekdayView.collapsed = this.collapsed;
     weekdayView.on('afterRender', function(weekdayViewModel) {
-        self.contentHeight = weekdayViewModel.minHeight + scheduleContainerTop;
+        self.contentHeight = weekdayViewModel.contentHeight + scheduleContainerTop;
+    });
+    weekdayView.on('sendMaxScheduleInDay', function(maxScheduleInDay) {
+        self.maxScheduleInDay = maxScheduleInDay;
```

Comment:
`maxScheduleInDay` is a variable of this class. I suggest to declare it first in constructor.